app/main/Concerns/Contains.php

```diff
@@ -8,11 +8,16 @@ trait Contains
 {
     protected function isJson(string $filename): bool
     {
-        return Str::endsWith($filename, '.json');
+        return Str::contains($filename, 'json');
```

Comment:
Can we use [str_contains](https://www.php.net/manual/en/function.str-contains) directly ?

Reply:
Exactly! Forgot about this feature. Thank you!

Yes, we can use it. My package contains this feature for earlier php versions.

I'll update the code in a few minutes.